scripts/deployment-handler.sh

```diff
@@ -95,6 +95,7 @@ function addCommonVariables(){
     writeCommonVariables "ProductVersion" "ProductVersion" true
     writeCommonVariables "S3AccessKeyID" "s3accessKey" true
     writeCommonVariables "S3SecretAccessKey" "s3secretKey" true
+    writeCommonVariables "TESTGRID_EMAIL_PASSWORD" "TESTGRID_EMAIL_PASSWORD" true
```

Comment:
if we are doing the 1st mentioned change, then the 2nd parameter should be changed to `testgridEmailPassword` too.